tests/Unit/Helpers/ParallelAlgorithms/Interpolation/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY "ParallelInterpolationHelpers")
+
+add_spectre_library(${LIBRARY} INTERFACE)
+
+spectre_target_headers(
+  ${LIBRARY}
+  INCLUDE_DIRECTORY ${CMAKE_SOURCE_DIR}/tests/Unit
+  HEADERS
+  Examples.hpp
```

Comment:
This won't compile the file, you'd have to move it out of Helpers/ into a cpp file.

Reply:
I get a build error though if I change something in this file though (like misspell a class name)

Reply:
Ah the file `tests/Unit/ParallelAlgorithms/Interpolation/Test_Protocols.cpp` is compiled and has the asserts.